<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgGestionEtapas</class>
 <widget class="QDialog" name="DlgGestionEtapas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Etapas</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="stagePanel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelEtapas">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Etapas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listEtapas">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="btnNuevaEtapa">
           <property name="text">
            <string>Nueva</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/resources/icons/new.svg</normaloff>:/icons/resources/icons/new.svg</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnEliminarEtapa">
           <property name="text">
            <string>Eliminar</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/resources/icons/remove.svg</normaloff>:/icons/resources/icons/remove.svg</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btnSubirEtapa">
           <property name="text">
            <string>Subir</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/resources/icons/chevron-up.svg</normaloff>:/icons/resources/icons/chevron-up.svg</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnBajarEtapa">
           <property name="text">
            <string>Bajar</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/resources/icons/chevron-down.svg</normaloff>:/icons/resources/icons/chevron-down.svg</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="detailsPanel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupEtapa">
         <property name="title">
          <string>Detalles de la Etapa</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelNombreEtapa">
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="edNombreEtapa">
            <property name="placeholderText">
             <string>Nombre de la etapa (obligatorio)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelCodigoEtapa">
            <property name="text">
             <string>Código:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="edCodigoEtapa">
            <property name="maxLength">
             <number>3</number>
            </property>
            <property name="placeholderText">
             <string>Código de 3 caracteres (obligatorio)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelDescripcionEtapa">
            <property name="text">
             <string>Descripción:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTextEdit" name="edDescripcionEtapa">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Descripción de la etapa (opcional)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupSubetapas">
         <property name="title">
          <string>Subetapas</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListView" name="listSubetapas">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QToolButton" name="btnNuevaSubetapa">
              <property name="text">
               <string>Nueva</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/icons/resources/icons/new.svg</normaloff>:/icons/resources/icons/new.svg</iconset>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnEliminarSubetapa">
              <property name="text">
               <string>Eliminar</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/icons/resources/icons/remove.svg</normaloff>:/icons/resources/icons/remove.svg</iconset>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="btnSubirSubetapa">
              <property name="text">
               <string>Subir</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/icons/resources/icons/chevron-up.svg</normaloff>:/icons/resources/icons/chevron-up.svg</iconset>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnBajarSubetapa">
              <property name="text">
               <string>Bajar</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/icons/resources/icons/chevron-down.svg</normaloff>:/icons/resources/icons/chevron-down.svg</iconset>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupDetallesSubetapa">
            <property name="title">
             <string>Detalles de la Subetapa</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="labelNombreSubetapa">
               <property name="text">
                <string>Nombre:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="edNombreSubetapa">
               <property name="placeholderText">
                <string>Nombre de la subetapa (obligatorio)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelDescripcionSubetapa">
               <property name="text">
                <string>Descripción:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QTextEdit" name="edDescripcionSubetapa">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>60</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Descripción de la subetapa (opcional)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelActivaSubetapa">
               <property name="text">
                <string>Activa:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="cbActivaSubetapa">
               <property name="text">
                <string>La subetapa está activa</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnCerrar">
       <property name="text">
        <string>Cerrar</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/resources/icons/circle-ok.svg</normaloff>:/icons/resources/icons/circle-ok.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
